x-pack/plugins/cross_cluster_replication/public/app/components/form_entry_row.js

```diff
@@ -77,6 +78,7 @@ export class FormEntryRow extends PureComponent {
             disabled={disabled === true}
             isLoading={isLoading}
             fullWidth
+            data-test-subj={dataTestSubj}
```

Comment:
...this line and line 93 would become `{...rest}`. Then the consumer of this component could just use `data-test-subj` as usual instead of needing to use `dataTestSubj`.